### PR TITLE
feat(hub): add temporary hub redirection

### PR DESCRIPTION
**Pull request by JeremyDec**
Signed-off-by: JeremyDec <jeremy.de-cesare@ovhcloud.com>

| Question         | Answer
| ---------------- | ---
| Branch?          | `master`
| Bug fix?         | no
| New feature?     | yes
| Breaking change? | no
| Tickets          | Fix #MANAGER-8725
| License          | BSD 3-Clause

- [x] Try to keep pull requests small so they can be easily reviewed.
- [x] Commits are signed-off
- [ ] ~~Only FR translations have been updated~~
- [x] Branch is up-to-date with target branch
- [x] Lint has passed locally
- [x] Standalone app was ran and tested locally
- [x] Ticket reference is mentioned in linked commits (internal only)
- [ ] ~~Breaking change is mentioned in relevant commits~~

### CODE REVIEW

**Review by Jisay — DISMISSED** [review]

**Comment by JeremyDec** [comment]
rebase on `develop` since last approval

File: packages/manager/apps/hub-redirection/package.json [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+    "dev:watch": "yarn run dev",
+    "start": "lerna exec --stream --scope='@ovh-ux/manager-hub-redirection-app' --include-dependencies -- npm run build --if-present",
+    "start:dev": "lerna exec --stream --scope='@ovh-ux/manager-hub-redirection-app' --include-dependencies -- npm run dev --if-present",
+    "start:watch": "lerna exec --stream --parallel --scope='@ovh-ux/manager-hub-redirection-app' --include-dependencies -- npm run dev:watch --if-present"
+  },
+  "dependencies": {},
+  "devDependencies": {
+    "@ovh-ux/manager-webpack-config": "^4.0.1",
+    "glob": "^7.1.6",
+    "lodash": "^4.17.15",
```
Comment (marie-j) [inline packages/manager/apps/hub-redirection/package.json:25]: Lodash needed ?

Comment (JeremyDec) [reply packages/manager/apps/hub-redirection/package.json:25]: Nope, can be removed :)

**Review by marie-j — APPROVED** [review]

File: packages/manager/apps/hub-redirection/webpack.config.js [hunk 2]
```diff
@@ -0,0 +1,73 @@
+const fs = require('fs');
+const glob = require('glob');
+const path = require('path');
+const webpack = require('webpack'); // eslint-disable-line
+const merge = require('webpack-merge');
+const webpackConfig = require('@ovh-ux/manager-webpack-config');
+
+function readNgAppInjections(file) {
```
Comment (frenautvh) [inline packages/manager/apps/hub-redirection/webpack.config.js:8]: you can remove many things from the config (to be honest i'm not even sure that using webpack is usefull in this case) :
- extras / app injections
- lessPath
- definePlugin
- resolve.modules

Comment (JeremyDec) [reply packages/manager/apps/hub-redirection/webpack.config.js]: In fact, dists generated with and without the webpack.config.js are different, so I prefer to change the webpack config file, to have the same with what as been done for former cloud application.

**Review by antleblanc — APPROVED** [review]
We can target `master` branch.

**Review by antleblanc — APPROVED** [review]

File: packages/manager/apps/hub-redirection/src/index.html [hunk 3]
```diff
@@ -0,0 +1,13 @@
+<!DOCTYPE html>
+<html lang="en">
+    <head>
+        <meta charset="utf-8" />
+        <title>OVHcloud</title>
+        <style type="text/css">
+            .ng-cloak {
```
Comment (marie-j) [inline packages/manager/apps/hub-redirection/src/index.html:7]: Non needed

**Review by marie-j — APPROVED** [review]

**Pull request merged**